worker/src/RTC/UdpSocket.cpp

```diff
@@ -10,20 +10,30 @@ namespace RTC
 {
 	/* Instance methods. */
 
-	UdpSocket::UdpSocket(Listener* listener, std::string& ip, RTC::Transport::SocketFlags& flags)
+	UdpSocket::UdpSocket(
+	  Listener* listener, std::string& ip, uint16_t port, RTC::Transport::SocketFlags& flags)
 	  : // This may throw.
-	    ::UdpSocketHandle::UdpSocketHandle(PortManager::BindUdp(ip, flags)), listener(listener)
+	    ::UdpSocketHandle::UdpSocketHandle(RTC::PortManager::BindUdp(ip, port, flags)),
+	    listener(listener), fixedPort(true)
 	{
 		MS_TRACE();
 	}
 
 	UdpSocket::UdpSocket(
-	  Listener* listener, std::string& ip, uint16_t port, RTC::Transport::SocketFlags& flags)
+	  Listener* listener,
+	  std::string& ip,
+	  uint16_t minPort,
+	  uint16_t maxPort,
+	  RTC::Transport::SocketFlags& flags,
+	  uint64_t& portRangeHash)
 	  : // This may throw.
-	    ::UdpSocketHandle::UdpSocketHandle(PortManager::BindUdp(ip, port, flags)), listener(listener),
-	    fixedPort(true)
```

Comment:
We had a critical BUG here! We marked UdpSocket with `fixedPort: true`, meaning that the port was never removed from `PortManager`'s maps when this UdpSocket was closed!!!